ppa/local_settings.py.sample

```diff
@@ -94,29 +94,29 @@ STATIC_ROOT = os.path.join(BASE_DIR, STATIC_URL.strip("/"))
 
 # Solution following https://stackoverflow.com/a/9541647
 # Sends a logging email even when DEBUG is on
-LOGGING = {
-    'version': 1,
-    'disable_existing_loggers': False,
-    'handlers': {
-        'mail_admins': {
-            'level': 'ERROR',
-            'class': 'django.utils.log.AdminEmailHandler',
-            'include_html': True
-        }
-    },
-    'loggers': {
-        'django.request': {
-            'handlers': ['mail_admins'],
-            'level': 'ERROR',
-            'propagate': True,
-        },
-        'SolrClient':  {
-            'handlers': ['console'],
-            'level': 'WARN'
-        },
-
-    }
-}
+# LOGGING = {
+#     'version': 1,
+#     'disable_existing_loggers': False,
+#     'handlers': {
+#         'mail_admins': {
+#             'level': 'ERROR',
+#             'class': 'django.utils.log.AdminEmailHandler',
+#             'include_html': True
+#         }
+#     },
+#     'loggers': {
+#         'django.request': {
+#             'handlers': ['mail_admins'],
+#             'level': 'ERROR',
+#             'propagate': True,
+#         },
+#         'SolrClient':  {
+#             'handlers': ['console'],
+#             'level': 'WARN'
+#         },
+# 
+#     }
+# }
```

Comment:
This throws errors if not configured properly and is not directly useful in a local environment, correct?

Reply:
The admin email handler isn't a logging config we'd want in a dev environment, but logging in general is quite useful and it would be nice if the local settings included a good default dev configuration.  Would you update the sample config with this one from mine?
```
LOGGING = {
    'version': 1,
    'disable_existing_loggers': True,
    'formatters': {
        'basic': {
            'format': '[%(asctime)s] %(levelname)s:%(name)s::%(message)s',
            'datefmt': '%d/%b/%Y %H:%M:%S',
        },
    },
    'handlers': {
        'console': {
            'class': 'logging.StreamHandler',
            'formatter': 'basic',
            'level': 'DEBUG'

        },
    },
    'loggers': {
        'django': {
            'handlers': ['console'],
            'level': os.getenv('DJANGO_LOG_LEVEL', 'WARN'),
        },
        'ppa':  {
            'handlers': ['console'],
            'level': 'DEBUG'
        },
        'SolrClient':  {
            'handlers': ['console'],
            'level': 'WARN'
        },
    }
}
```